cmd/power/power.go

```diff
@@ -19,6 +21,53 @@ const (
 	DOCKER_CGROUPS_FILE = "/proc/self/cgroup"
 )
 
+func PowerMain() {
+	app := cli.NewApp()
+
+	app.Name = os.Args[0]
+	app.Version = config.VERSION
+	app.Author = "Rancher Labs, Inc."
+	app.Email = "sid@rancher.com"
```

Comment:
Please don't put your personal email.  Git commits/blame are better to determine authors